index.html

```diff
@@ -0,0 +1,115 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>판다마켓</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="/"><img src="img/logo.png" /></a>
+        <a href="/login"> <button>로그인</button></a>
```

Comment:
❗️ 수정요청
로그인 버튼을 눌렀을 때 './login.html'로 이동시켜주는 역할을 하니, 앵커 태그만 남겨두셔도 될 것 같습니다~
버튼 요소처럼 보여도 꼭 버튼 태그로 작성하실 필요는 없습니다!
해당 태그의 역할에 따라 태그를 작성하시는 것이 더 시맨틱하게 작성하는 방법입니다.

또한 지금 단계에서 페이지 이동을 구현하시기 위해서는 확장자를 포함시켜주셔야 합니다.
관련 [이슈](https://github.com/tauri-apps/tauri/issues/2328)를 첨부해드리니 한번 확인해보시면 좋을 것 같습니다.

```suggestion
        <a href="/login.html">로그인</a>
```

Reply:
💬 여담
구현을 완료하신 후에는 작업하신 사항들이 잘 구현되엇는지, a tag, button tag들을 눌러보시면서 확인하시면 좋습니다~